languages/ruby/reference/exercise-concepts/resistor-color-duo.md

```diff
@@ -0,0 +1,49 @@
+# Concepts of Resistor Color Duo
+
+## General
+
+ [Example solution](https://github.com/exercism/ruby/blob/master/exercises/resistor-color-duo/.meta/solutions/resistor_color_duo.rb)
+
+- Classes:
+  - Defining classes
+- Methods:
+  - Defining methods
+  - Arguments
+  - Return values
+- Strings:
+  - Basics like difference between `1` and `"1"`
+- Class methods:
+  - How to define a method so it can be called directly on a class
+
+Depending on the approach either:
+- Hashes:
+  - How to initialize/create a hash
+  - How to access it's values
+
+or
+- Arrays:
+   - How to initialize/create an array
+   - What are indexes and how can they be useful
+   - How to get the index of a value that's stored in an array
+
+### Approach: use string interpolation (positional approach)
+
+- String interpolation ( `#{}` )
+- Transforming strings into integers ( `#to_i` )
+
+### Approach: Using enumerable methods
+
+- Loops:
+   - What a loop is and how it works
+   - Enumerable methods and finding the appropriate one
+   - Return value of enumerable methods (looping with `#each` and `#map`)
```

Comment:
`each` is not an `Enumerable` method.  Maybe consider not listing any methods.  Part of discovery process.  Instead, perhaps link to the documentation?